bls/src/keypair.rs

```diff
@@ -229,4 +359,19 @@ mod tests {
         assert_eq!(keypair.secret, secret);
         assert_eq!(keypair.public, public);
     }
+
+    #[test]
+    #[cfg(feature = "fs")]
+    fn test_keypair_file() {
+        let out_dir = std::env::var("FARF_DIR").unwrap_or_else(|_| "farf".to_string());
+        let outfile = format!("{}/tmp/test_write_keypair_file.json", out_dir);
```

Comment:
nit: It'd be better to use `tempfile` for these  rather than the `FARF` directory